pkg/zng/docs/spec.md

```diff
@@ -407,7 +427,7 @@ Except for typedefs, all control messages have the form
 ```
 #!<control code>:<payload>
 ```
-where <control code> is a decimal integer in the range 5-127 and <payload>
+where `<control code>` is a decimal integer in the range 5-127 and `<payload>`
```

Comment:
The fixed width here is not just for formatting purposes. When written "bare" these seem to get swallowed up by Markdown, presumably because they're mistaken for HTML.